pseudomonas_analysis/2_identify_generic_genes_pathways.py

```diff
@@ -291,6 +292,37 @@
 ]
 generic_SOPHIE_only.to_csv(os.path.join(local_dir, "SOPHIE_generic_only.tsv"), sep="\t")
 
+shared_ranking.head()
+
+# Hypergeometric test:
+# Given N number of genes with K common genes in GAPE.
+# SOPHIE identifies n genes as being common
+# What is the probability that k of the genes identified by SOPHIE
+# are also common in GAPE? What is the probability of drawing
+# k or more concordant genes?
+num_GAPE_genes = shared_ranking.shape[0]
+num_generic_GAPE_genes = shared_ranking[shared_ranking[ref_rank_col] >= 60.0].shape[0]
```

Comment:
Why is this threshold different (60% instead of 80%)?

Reply:
The 60% is based on a manual visual inspection of the correlation plot

Reply:
This question revealed a bug! Turns out that I didn't provide the proper input for this comparison so the distribution wasn't uniform as it should be. Now that its fixed, the threshold should be 80 in both cases:)

Reply:
Good to hear!